### DISCUSSION TITLE
Release v1.8.3

### DISCUSSION THREAD

**Discussion by hhyo** [hhyo/Archery#1423] [General]
## Release date: 2022年03月27日

代码变更: https://github.com/hhyo/Archery/compare/v1.8.2...v1.8.3
贡献者: @AceAttorney, @CrusM, @LeoQuote, @blylei, @cpzt, @dependabot, @dependabot[bot], @heisaman, @hhyo, @issacmark, @lijingeng, @nick2wang, @ningyu1, @ocpeng, @quanbisen, @unknowissue and @weideguo

## 新增
- 支持ClickHouse上线、查询 @nick2wang (#1384 #1403)
- 支持ODPS查询 @cpzt (#1363)
- 新增独立的审计模块 @weideguo (#1251 #1342)
- 新增my2sql工具插件模块 #1224 @nick2wang (#1314)
- 新增mssql、oracle的数据字典支持  (#1351 #1402)
- mysql账号管理支持回收权限(REVOKE) @nick2wang (#1341)
- redis支持集群模式 @nick2wang (#1392)

## 优化
- SQL工单通知支持分阶段控制 @nick2wang (#1406)
- 增加数据查询下载的权限(hhyo#1367) @weideguo (#1369)
- 为ajax请求的返回错误进行分类报错 (hhyo#1346) @weideguo (#1365)
- Archery Kubernetes Helm Chart 维护 @AceAttorney (#1334)
- oracle执行DDL时备份原始定义 @unknowissue (#1332)
- oracle查看表结构增加注释列 @unknowissue (#1320)
- oracle脱敏更换函数 @unknowissue (#1306)
- 对实例列表进行排序(支持汉字排序) #1290 @nick2wang (#1322)
- 添加redis/pg创建连接时的超时参数 (hhyo#1311) @weideguo (#1321)
- redis查询结果json格式化显示 #1376 @nick2wang (#1382)
- SSH隧道连接新增选择本地文件上传密钥，上传后将密钥信息保存到数据库，连接时从数据库读取 @issacmark (#1303)
- SQL上线时可执行时间范围前端优化，要求时间间隔必须大于60分钟 (hhyo#1259) @weideguo (#1279)

## 修复
- 修复实例检测返回连接状态错误bug fix #1179 @nick2wang (#1407)
- 修复mongodb表名带-和.符号无法申请上线的问题 @CrusM (#1372 #1381)
- 修复调用shell执行mongodb命令时出现WARNING的问题 (hhyo#1237) @weideguo (#1348)
- 修复插件shell命令注入漏洞 @nick2wang (#1316)
- 修复Redis hgetall结果不展示的问题 @cpzt (#1315)
- 修复手动执行完成的工单没有发送通知的问题#1284 @nick2wang (#1304)
- 修复缩略SQL乱码问题 @ocpeng (#1305)
- 修复会话管理存在的sql注入风险 #1299 @nick2wang (#1300)
- 修复创建mongodb原生连接时的错误 (hhyo#1291) @weideguo (#1294)
- 修复pgsql获取表结构没有增加schame过滤的问题 @lijingeng (#1285)
- 修复admin.sh中SQL路径错误的问题 @blylei (#1276)

## 升级步骤
- https://github.com/hhyo/archery/wiki/upgrade
- 本次升级涉及较多的数据模型变更，升级前请注意数据备份
- ssh隧道管理连接存在不兼容的变更，请参考：https://github.com/hhyo/Archery/pull/1303#discussion_r779257350

<hr /><em>This discussion was created from the release <a href='https://github.com/hhyo/Archery/releases/tag/v1.8.3'>Release v1.8.3</a>.</em>

**Comment by LeoQuote** [comment]
应该直接来个 1.9.0 的
